Shared/Samples/Validate utility network topology/ValidateUtilityNetworkTopologyView.swift

```diff
@@ -0,0 +1,190 @@
+                .onSingleTapGesture { screenPoint, _ in
+                    selectedOperation = .selectFeature(screenPoint: screenPoint)
+                }
+                .task(id: selectedOperation) {
+                    operationIsRunning = true
+                    defer { operationIsRunning = false }
+                    
+                    do {
+                        switch selectedOperation {
+                        case .setup:
+                            try await model.setup()
+                            
+                        case .getState:
+                            try await model.getState()
+                            
+                        case .trace:
+                            try await model.trace()
+                            
+                        case .validateNetworkTopology:
+                            guard let extent = visibleArea?.extent else { return }
+                            try await model.validate(forExtent: extent)
+                            
+                        case .selectFeature(let screenPoint):
+                            // Identify the tapped layers using the map view proxy.
+                            let identifyResults = try await mapViewProxy.identifyLayers(
+                                screenPoint: screenPoint!,
+                                tolerance: 5
+                            )
+                            model.selectFeature(from: identifyResults)
+                            
+                            if model.feature != nil {
+                                // Present the sheet to edit the feature if one was selected.
+                                editSheetIsPresented = true
+                            } else {
+                                model.statusMessage = "No feature identified. Tap on a feature."
+                            }
+                            
+                        case .applyEdits:
+                            try await model.applyEdits()
+                            
+                        case .clearSelection:
+                            model.clearSelection()
+                            model.statusMessage = "Selection cleared."
+                        }
+                    } catch {
+                        model.statusMessage = selectedOperation.errorMessage
+                        self.error = error
+                    }
+                }
+                .task(id: editSheetIsPresented) {
+                    guard editSheetIsPresented,
+                          let featureCenter = model.feature?.geometry?.extent.center,
+                          let visibleArea else { return }
+                    
+                    // Set the map's viewpoint to center the feature in the top half of the screen.
+                    let yOffset = (visibleArea.extent.height / 2) / 2
+                    let offsettedCenter = Point(x: featureCenter.x, y: featureCenter.y - yOffset)
```

Comment:
Perhaps consider using offset as an adjective instead so `offsetCenter`.

```suggestion
                    let offsetCenter = Point(x: featureCenter.x, y: featureCenter.y - yOffset)
```

Or, even better look into using [`contentInsets(_:)`](https://developers.arcgis.com/swift/api-reference/documentation/arcgis/mapview/contentinsets(_:)) and I think you might be able to do away with this entire task.